nixos/modules/programs/nix-ld.nix

```diff
@@ -0,0 +1,11 @@
+{ pkgs, lib, config, ... }:
+{
+  options = {
+    programs.nix-ld.enable = lib.mkEnableOption "nix-ld";
```

Comment:
I can drop also some paragraphs in the manual. Any pointers where this could go?

Reply:
I guess this would be best off in the "Administration" section of the NixOS manual, maybe just before the "Troubleshooting" chapter?

Reply:
Added changelog.

Reply:
Linked to upstream documentation. This one also has nix examples.